string load
string unload
string go_start
---
string[] load_directions
string[] unload_directions
string[] go_start